.github/workflows/slither-report.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Check slither report
         run: |
           section=$(awk '/^##/{exit} {print}' slitherReport.md)
-          if [[ $section == *"Medium"* || $section == *"High"* ]]; then
-            echo "Medium or High risk found in slither report. Failing the run."
+          if [[ $section == *"Medium"* || $section == *"High"* || $section == *"Low"* ]]; then
```

Comment:
Clean up ordering so it's Low -> Medium -> High here

Reply:
As in like the visual? The ors mean it will fail if any of Low, Medium or High are detected